start.py

```diff
@@ -1,13 +1,43 @@
+import os
 import sys
+import __main__
 
 from PyQt5.QtWidgets import (QApplication, QMessageBox, QSystemTrayIcon)
 from osfoffline.application.main import OSFApp
 from osfoffline import utils
 
 
+def is_only_instance():
+    # Determine if there are more than the current instance of the application
+    # running at the current time.
+    return os.system("(( $(ps -ef | grep python | grep '[" +
+                     __main__.__file__[0] + "]" + __main__.__file__[1:] +
+                     "' | wc -l) > 1 ))") != 0
```

Comment:
Will this work when using a pyinstaller-bundled application?

If we think the mac fix is going to be tricky, one possibility would be to limit this fix to Windows-only for now (since that is our focus for the alpha release), and make a ticket (pointing to this commit) so that we can revisit mac error messages in the future.